src/cmds/asset.js

```diff
@@ -1,7 +1,9 @@
 const path = require('path')
 const uploadFileOrDirectory = require('../helpers/uploadFileOrDirectory')
+const upgradeCliVersion = require('./upgrade')
 
 module.exports = function asset() {
+  upgradeCliVersion()
```

Comment:
Also I think, there should be 2 distincts function. One dedicated to check if a new version can be applied and the second dedicated to do the upgrade without asking nothing. Currently it will promp a message by calling `livestorm upgrade` and I don't think we want that.

And the functions should be blocking, so that the initial command can be executed after the upgrade.

Example: 
```
const isCandidateForUpgrade = await checkCandidateForUpgrade() (check periodicity and version)
if (isCandidateForUpgrade) {
    await promptUpgrade()
}
availableCommand(...)
```

Wdyt ?